### PR TITLE
feat: added different types of text format to CL

### PR DESCRIPTION
**Pull request by ajit-sharmaGH**
image

### CODE REVIEW

**Review by paarthbajaj — APPROVED** [review]
Looks good. I liked how you have made good utility classes. It will be good for your future projects. You can merge.

File: CSS/base/color.css [hunk 1]
```diff
@@ -31,4 +31,27 @@
...
+    --gray: #D1D5DB;
+    --dark-gray: #4B5563;
+    --light-gray: #E5E7EB;
+    --darker-gray: #212529;
+    --primary-complimentary: #fcc5c5;
+    --golden: #fdc52c;
+    --dark-mode-bg: #181733;
+    --dark-bg-text: hsla(0, 0%, 100%, .75);
+    --off-white-hover: #E5E7EB;
+    --cornsilk: cornsilk;
```
Comment (Nagashree01) [inline CSS/base/color.css:51]: you can use hex code for cornsilk color and wheat color as all other colors are in hex code

**Review by Nagashree01 — APPROVED** [review]
Just left one suggestion. Code is clean and good to merge

File: CSS/base/color.css (hunk 1 above)
Comment (ajit-sharmaGH) [reply CSS/base/color.css:51]: sure will do that ! thank you !

**Pull request merged**